.github/workflows/build-magma-rocm-linux.yml

```diff
@@ -0,0 +1,69 @@
+name: build-linux-magma-rocm
+
+on:
+  push:
+    branches:
+      main
+    paths:
+      - .ci/magma-rocm/*
+      - .ci/magma-rocm/package_files/*
+      - .github/workflows/build-magma-rocm-linux.yml
+  pull_request:
+    paths:
+      - .ci/magma-rocm/*
+      - .ci/magma-rocm/package_files/*
+      - .github/workflows/build-magma-rocm-linux.yml
+
+defaults:
+  run:
+    shell: bash -x -e -l {0}
+env:
+  BUILD_ENVIRONMENT: build-linux-magma-rocm
+  IN_CI: 1
+  IS_GHA: 1
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.sha }}-${{ github.event_name == 'workflow_dispatch' }}
+  cancel-in-progress: true
+
+jobs:
+  build-linux-magma-rocm:
+    if: github.repository_owner == 'pytorch'
+    runs-on: linux.2xlarge
```

Comment:
Why not 12x large (magma updates should be pretty infrequent) Also, it must be ephemeral...

Reply:
Done.